src/engine/source/hlp/CMakeLists.txt

```diff
@@ -26,6 +17,7 @@ CPMAddPackage(
         "RAPIDJSON_BUILD_DOC OFF"
         "RAPIDJSON_BUILD_EXAMPLES OFF"
         "RAPIDJSON_BUILD_TESTS OFF"
+        "RAPIDJSON_HAS_STDSTRING ON"
```

Comment:
This was an artifact from an earlier test and can be removed

Reply:
done